client.go

```diff
@@ -292,7 +292,7 @@ type DataPoint struct {
 	Value     float64
 }
 
-func parseDataPoints(info interface{}) (dataPoints []DataPoint, err error) {
+func ParseDataPoints(info interface{}) (dataPoints []DataPoint, err error) {
```

Comment:
@filipecosta90 I don't object of exposing this, but I missing the reasoning of doing that.

Reply:
@danni-m I think the main reason is to decouple the client logic with response parsing logic. If the user wants to use their client and parse the response with our implementation they can ( we need it for TS.RANGE parsing ).